notebooks/Notebook3.md

```diff
@@ -9,6 +9,19 @@ Did it help you make a design decision? Was it helpful in addressing the most pr
 issues in your project? How will you incorporate the feedback into your work? Will you
 change something about the design, implementation, or evaluation as a result?
 
+There was some advice on reducing verbosity. For example, I had wanted to create a pattern with
+this synatx:
+
+```{"Pattern a"}
+don don don
+```
+
+A simple solution that Chris pointed out was to just have 1 backtick instead of 3. It's a very
+minor adjustment, but that's 4 less keystrokes one would have to do so I think I'll incorporate that!
+
+I also had an issue where there wasn't always a name for a specific note. For instance, we have "don" for 
+a quarter note, and "do" for an eighth, but none for a sixteenth. Gabriel suggested making up a name for sixteenth notes, but I feel like the issue isn't just specifically the 16th note. There may be a bit of a slippery slope where we just start making up a lot of syntax that people are not necessarily familiar with which defeats the point of having the language be similar to what people already use. For that reason, I'd like to stick with this format for now. 
```

Comment:
Another random idea — feel free to reject if you don't think this would make sense for your users: you could let *them* define custom notes? For instance, each input file could have an (optional?) header section before the main section where they could write things like
```
si = do[1/2]
```
and then in the main section wherever they write `si` it gets interpreted as a sixteenth note?